src/auto/cell_renderer_accel.rs

```diff
@@ -1,9 +1,12 @@
-// This file was generated by gir (4d68d19) from gir-files (11e0e6d)
+// This file was generated by gir (bc9cb02) from gir-files (???)
```

Comment:
Its rust's git bug with submodule `objects/info/alternates` in msys.
I make it normal when ready to merge.